assets/css/grammar.css

```diff
@@ -1,49 +1,16 @@
-/* no line numbering in syntax outlines */
-.syntax-outline .code-lines li::before {
-    content: ' ';
-}
-.syntax-outline .code-lines {
-    line-height: 1.9em;
-}
-.syntax-outline div.Swift {
-    border-color: transparent;
-}
-
-.code-sample div.Swift {
-    width: 93%;
-    margin-left: 13px;
-    overflow: visible;
-}
-div.Swift {
-    border-left: 5px solid rgba(205,233,244,1);
-    padding: 4px 0 2px 10px;
-    margin: 10px 0 21px;
-    line-height: 22px;
-}
-
-.highlight .gi { /* code placeholder capsule */
-    background-color: rgba(233, 239, 250, 1);
-    border: 1px solid rgba(196, 212, 241, 1);
-    border-radius: 10px;
-    color: rgba(0, 0, 0, 1);
-    padding: 2px 6px;
-    font-style: inherit;
-    white-space: nowrap
-}
-
-
```

Comment:
Why all this deletion?

Reply:
When I created grammar.css (which is now too narrowly named), I created some classes, etc., that I never actually used.   I removed them, now, to eliminate clutter and unused code.

Reply:
Ah, I see.  I wrongly thought this was the main .css file.